include/fastdds/dds/core/policy/QosPolicies.hpp

```diff
@@ -23,6 +23,7 @@
 #include <vector>
 
 #include <fastdds/dds/core/policy/ParameterTypes.hpp>
+#include <fastdds/dds/xtypes/type_representation/detail/dds_xtypes_typeobject.hpp>
 #include <fastdds/rtps/attributes/ExternalLocators.hpp>
 #include <fastdds/rtps/attributes/PropertyPolicy.h>
 #include <fastdds/rtps/attributes/RTPSParticipantAllocationAttributes.hpp>
```

Comment:
To ensure that the migration has been correctly completed, please remove the old header `fastrtps/types/TypeObject.h`.

Reply:
Removed in [83756a6](https://github.com/eProsima/Fast-DDS/pull/4140/commits/83756a6445a363860a622ce24e1cbea2985a6c6a)